app/assets/stylesheets/styles.scss

```diff
@@ -2702,7 +2776,7 @@ body.dark-theme {
     #topNavbar .logo-link .logo-nb {
         filter: invert(100%);
     }
-    .language-icon {
+    .language-icon[href*="/languages/python"] {
```

Comment:
Why only for languages/python?

Reply:
For the record, the reason is so that the Python logo's # (2/3) is visible in display modes with dark backgrounds.